protocol-gen/package.json

```diff
@@ -17,7 +17,7 @@
     "typescript": "^4.2.3"
   },
   "dependencies": {
-    "@dcl/protocol": "^1.0.0-3250759085.commit-6cf3623",
+    "@dcl/protocol": "https://sdk-team-cdn.decentraland.org/@dcl/protocol/branch//dcl-protocol-1.0.0-3267930139.commit-2b6939d.tgz",
```

Comment:
Is this expected?

Reply:
Yes, I will actually update that dependency again after we merge the [Protocol PR](https://github.com/decentraland/protocol/pull/63), meanwhile this is poiting to the package build in that PR.